hugo/content/contributors/jeff-delaney.md

```diff
@@ -5,7 +5,7 @@ draft: false
 featured_img: /img/contributors/jeff-delaney.png
 
 links: 
-    youtube: https://www.youtube.com/channel/UCsBjURrPoezykLs9EqgamOA
+    youtube: https://www.youtube.com/c/Fireship/
```

Comment:
The URL that you have changed is working fine as youtube uses URLs like ```channel/:channel_Id``` or  ```c/:channel_name```, both are the right syntax.

But URL that you have specified in the conversation is misguiding.

Anyways this change is fine as it is more readable. ✌️ 

Anyways, this url